lib/aullar/flair.moon

```diff
@@ -35,8 +35,22 @@ set_line_type_from_flair = (cr, flair) ->
       cr.dash = {6, 3}
 
 draw_ops = {
-  rectangle: (flair, x, y, width, height, cr) ->
+  custom: (flair, x, y, width, height, cr) ->
+    custom_fn = flair.custom_draw
+    if custom_fn and pcall custom_fn, flair, x, y, width, height, cr
```

Comment:
You're throwing away the actual error message here which would make it hard to understand why it's failing. For a failed invocation the error message should be visible somehow - either by printing it to stderr or by failing the draw completely and let the default handling take care of it.